### ISSUE TITLE
h264 file with 4:4:4 predictive mode also does not work

### ISSUE THREAD

**Issue by Randrianasulu** [heroineworshiper/hvirtual#11]
```
[root@slax hvirtual]# bin/cinelerra /dev/shm/1.mp4
Cinelerra 9 (C)2024 Adam Williams

Cinelerra is free software, covered by the GNU General Public License,
and you are welcome to change it and/or distribute copies of it under
certain conditions. There is absolutely no warranty for Cinelerra.
BC_DisplayInfo::test_window 158: No window manager
EditPanel::create_objects: meter_panel == 0
quicktime_new_ffmpeg 202 cpus=4 codec=h264 id=27
quicktime_new_ffmpeg 202 cpus=4 codec=h264 id=27
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 9 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
EditPanel::create_objects: meter_panel == 0
quicktime_new_ffmpeg 202 cpus=4 codec=h264 id=27
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
[h264 @ 0x7faa68463300] Invalid NAL unit size (670 > 9).
[h264 @ 0x7faa68463300] Error splitting the input into NAL units.
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
[h264 @ 0x7faa681c9940] Invalid NAL unit size (670 > 9).
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
[h264 @ 0x7faa681c9940] Error splitting the input into NAL units.
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
decode_wrapper 501: unrecognized color model 14
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
[h264 @ 0x7faa68209e40] Invalid NAL unit size (670 > 9).
[h264 @ 0x7faa68209e40] Error splitting the input into NAL units.
[h264 @ 0x7faa68394600] Invalid NAL unit size (670 > 9).
[h264 @ 0x7faa68394600] Error splitting the input into NAL units.
cmodel_transfer 508 unsupported conversion 0 -> 14 bg_color=00000000
cmodel_transfer 513 out=0,0 320,240 in=0,0 320,240
```
```
[root@slax hvirtual]# file /dev/shm/1.mp4
/dev/shm/1.mp4: ISO Media, MP4 Base Media v1 [IS0 14496-12:2003]
[root@slax hvirtual]# ffprobe /dev/shm/1.mp4
ffprobe version 4.1.6 Copyright (c) 2007-2020 the FFmpeg developers
  built with gcc 5.5.0 (Linaro GCC 5.5-2017.10) 20171010 (ROSA)
  configuration: --prefix=/usr --enable-shared --libdir=/usr/lib64 --shlibdir=/usr/lib64 --incdir=/usr/include --disable-stripping --enable-postproc --enable-gpl --enable-pthreads --enable-ladspa --enable-libbluray --enable-libtheora --enable-libvorbis --disable-encoder=vorbis --enable-libvpx --enable-libxcb --enable-libxcb-shm --enable-libxcb-xfixes --enable-libxcb-shape --enable-runtime-cpudetect --enable-libfdk-aac --enable-libdc1394 --enable-librtmp --enable-libspeex --enable-libfreetype --enable-libgsm --enable-libcelt --enable-avresample --enable-opencl --enable-nvdec --enable-nvenc --disable-libopencv --enable-libopenjpeg --enable-libopus --enable-libsoxr --enable-libwavpack --disable-libxavs --enable-libmodplug --enable-libmp3lame --enable-libass --enable-gnutls --enable-libcdio --enable-libpulse --enable-libv4l2 --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-version3 --enable-libx264 --enable-libx265 --enable-libvo-amrwbenc --enable-libxvid
  libavutil      56. 22.100 / 56. 22.100
  libavcodec     58. 35.100 / 58. 35.100
  libavformat    58. 20.100 / 58. 20.100
  libavdevice    58.  5.100 / 58.  5.100
  libavfilter     7. 40.101 /  7. 40.101
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  3.100 /  5.  3.100
  libswresample   3.  3.100 /  3.  3.100
  libpostproc    55.  3.100 / 55.  3.100
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '/dev/shm/1.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    encoder         : Lavf60.16.100
  Duration: 00:00:03.00, start: 0.000000, bitrate: 16 kb/s
    Stream #0:0(und): Video: h264 (High 4:4:4 Predictive) (avc1 / 0x31637661), yuvj444p(pc, bt709/unknown/unknown), 320x240 [SAR 1:1 DAR 4:3], 11 kb/s, 25 fps, 25 tbr, 12800 tbn, 50 tbc (default)
    Metadata:
      handler_name    : VideoHandler
```
You basically should be able to make such file by just ending ffmpeg line reading 444p y4m  with file ending with .mp4, so default x264 encoder will kick in.

```
ffmpeg -i /dev/shm/yuv-mpeg-yuv444p.y4m /dev/shm/1.mp4
ffmpeg version 4.4.5 Copyright (c) 2000-2024 the FFmpeg developers
  built with gcc 11.2.0 (GCC)
  configuration: --prefix=/usr --libdir=/usr/lib --shlibdir=/usr/lib --docdir=/usr/doc/ffmpeg-4.4.5/html --mandir=/usr/man --disable-debug --enable-shared --disable-static --enable-gpl --enable-version3 --enable-avresample --arch=i586 --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-gnutls --enable-libass --enable-libbluray --enable-libcdio --enable-frei0r --enable-libgsm --enable-openal --enable-libopus --enable-librtmp --enable-libsnappy --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libv4l2 --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --enable-libmp3lame --enable-opencl --enable-opengl --enable-libopenjpeg --disable-libpulse --enable-libsmbclient --enable-libsvtav1 --enable-libxml2 --enable-librsvg --enable-libdrm --enable-libaom --enable-libdav1d --enable-libsoxr --enable-libzimg --enable-vapoursynth
  libavutil      56. 70.100 / 56. 70.100
  libavcodec     58.134.100 / 58.134.100
  libavformat    58. 76.100 / 58. 76.100
  libavdevice    58. 13.100 / 58. 13.100
  libavfilter     7.110.100 /  7.110.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  9.100 /  5.  9.100
  libswresample   3.  9.100 /  3.  9.100
  libpostproc    55.  9.100 / 55.  9.100
Input #0, yuv4mpegpipe, from '/dev/shm/yuv-mpeg-yuv444p.y4m':
  Duration: 00:00:00.08, start: 0.000000, bitrate: 46087 kb/s
  Stream #0:0: Video: rawvideo (444P / 0x50343434), yuv444p(pc, progressive), 320x240, SAR 1:1 DAR 4:3, 25 fps, 25 tbr, 25 tbn, 25 tbc
File '/dev/shm/1.mp4' already exists. Overwrite? [y/N] y
Stream mapping:
  Stream #0:0 -> #0:0 (rawvideo (native) -> h264 (libx264))
Press [q] to stop, [?] for help
[libx264 @ 0x9851540] using SAR=1/1
[libx264 @ 0x9851540] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX XOP FMA3 BMI1
[libx264 @ 0x9851540] profile High 4:4:4 Predictive, level 1.3, 4:4:4, 8-bit
[libx264 @ 0x9851540] 264 - core 163 - H.264/MPEG-4 AVC codec - Copyleft 2003-2021 - http://www.videolan.org/x264.html - options: cabac=1 ref=3 deblock=1:0:0 analyse=0x3:0x113 me=hex subme=7 psy=1 psy_rd=1.00:0.00 mixed_ref=1 me_range=16 chroma_me=1 trellis=1 8x8dct=1 cqm=0 deadzone=21,11 fast_pskip=1 chroma_qp_offset=4 threads=6 lookahead_threads=1 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=3 b_pyramid=2 b_adapt=1 b_bias=0 direct=1 weightb=1 open_gop=0 weightp=2 keyint=250 keyint_min=25 scenecut=40 intra_refresh=0 rc_lookahead=40 rc=crf mbtree=1 crf=23.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 ip_ratio=1.40 aq=1:1.00
Output #0, mp4, to '/dev/shm/1.mp4':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Video: h264 (avc1 / 0x31637661), yuv444p(pc, progressive), 320x240 [SAR 1:1 DAR 4:3], q=2-31, 25 fps, 12800 tbn
    Metadata:
      encoder         : Lavc58.134.100 libx264
    Side data:
      cpb: bitrate max/min/avg: 0/0/0 buffer size: 0 vbv_delay: N/A
frame=    1 fps=0.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   frame=    2 fps=0.0 q=-1.0 Lsize=       2kB time=00:00:00.04 bitrate= 370.3kbits/s speed=0.686x
video:1kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 88.133873%
[libx264 @ 0x9851540] frame I:1     Avg QP:24.00  size:   279
[libx264 @ 0x9851540] frame P:1     Avg QP:26.00  size:    33
[libx264 @ 0x9851540] mb I  I16..4: 68.0% 32.0%  0.0%
[libx264 @ 0x9851540] mb P  I16..4:  0.7%  0.7%  0.0%  P16..4:  3.7%  0.0%  0.0%  0.0%  0.0%    skip:95.0%
[libx264 @ 0x9851540] 8x8 transform intra:32.2%
[libx264 @ 0x9851540] coded y,u,v intra: 3.3% 0.0% 0.0% inter: 0.0% 0.0% 0.0%
[libx264 @ 0x9851540] i16 v,h,dc,p: 90%  9%  0%  0%
[libx264 @ 0x9851540] i8 v,h,dc,ddl,ddr,vr,hd,vl,hu: 90%  0% 10%  0%  0%  0%  0%  0%  0%
[libx264 @ 0x9851540] Weighted P-Frames: Y:0.0% UV:0.0%
[libx264 @ 0x9851540] kb/s:31.20
```

https://github.com/user-attachments/assets/989f08d2-34c0-4b26-a6b8-dd318ad67946

**Comment by Randrianasulu** [comment]
I think this one on top of h264 fix already in git needed

```
diff --git a/quicktime/qtffmpeg.c b/quicktime/qtffmpeg.c
index 4c647d41..c9c66680 100644
--- a/quicktime/qtffmpeg.c
+++ b/quicktime/qtffmpeg.c
@@ -487,10 +487,12 @@ static int decode_wrapper(quicktime_t *file,
     //                         break;

             case AV_PIX_FMT_YUV444P:
+            case AV_PIX_FMT_YUVJ444P:
                            file->src_colormodel = BC_YUV444P;
                 break;

                    case AV_PIX_FMT_YUV422P:
+                   case AV_PIX_FMT_YUVJ422P:
                            file->src_colormodel = BC_YUV422P;
                            break;
                    case AV_PIX_FMT_YUV410P:
```

basically J variants are deprecated (but still returnable by decoder in ffmpeg 5.1 ?) pixel formats with full chroma levels 

```
  AV_PIX_FMT_YUVJ420P,  ///< planar YUV 4:2:0, 12bpp, full scale (JPEG), deprecated in favor of AV_PIX_FMT_YUV420P and setting color_range
    AV_PIX_FMT_YUVJ422P,  ///< planar YUV 4:2:2, 16bpp, full scale (JPEG), deprecated in favor of AV_PIX_FMT_YUV422P and setting color_range
    AV_PIX_FMT_YUVJ444P,  ///< planar YUV 4:4:4, 24bpp, full scale (JPEG), deprecated in favor of AV_PIX_FMT_YUV444P and setting color_range
```

**Comment by Randrianasulu** [comment]
fixed at least in c1d4250a29e7fed632a9ec6a5fab2dec0d63a437, thanks!
